src/main/kotlin/org/rust/ide/annotator/RsLiteralAnnotator.kt

```diff
@@ -26,13 +25,15 @@ class RsLiteralAnnotator : AnnotatorBase() {
                 val suffix = literal.suffix
                 val validSuffixes = literal.validSuffixes
                 if (!suffix.isNullOrEmpty() && suffix !in validSuffixes) {
-                    holder.createErrorAnnotation(element, if (validSuffixes.isNotEmpty()) {
+                    val message = if (validSuffixes.isNotEmpty()) {
                         val validSuffixesStr = validSuffixes.map { "'$it'" }.joinToString()
                         "invalid suffix '$suffix' for ${literal.node.displayName}; " +
                             "the suffix must be one of: $validSuffixesStr"
                     } else {
                         "${literal.node.displayName} with a suffix is invalid"
-                    })
+                    }
+
+                    holder.newAnnotation(HighlightSeverity.ERROR, message).range(element).create()
```

Comment:
Note, `range(element)` is useless when `element` if the same as parameter passed to `annotateInternal` because new API adds the corresponding range by default

Reply:
This is indeed the case. Let me modify it, although I think a clear statement of scope may be better.